File: event/src/test/java/de/danoeh/antennapod/event/settings/VolumeAdaptionChangedEventTest.java
package de.danoeh.antennapod.event.settings;

import org.junit.Test;

import de.danoeh.antennapod.model.feed.VolumeAdaptionSetting;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertSame;

public class VolumeAdaptionChangedEventTest {

    @Test
    public void volumeAdaptionSettingAndFeedIdAreStored() {
        VolumeAdaptionChangedEvent event =
                new VolumeAdaptionChangedEvent(VolumeAdaptionSetting.LIGHT_REDUCTION, 3L);
        assertSame(VolumeAdaptionSetting.LIGHT_REDUCTION, event.getVolumeAdaptionSetting());
        assertEquals(3L, event.getFeedId());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/settings/VolumeAdaptionChangedEventTest.kt
package de.danoeh.antennapod.event.settings

import de.danoeh.antennapod.model.feed.VolumeAdaptionSetting
import org.junit.Assert.assertEquals
import org.junit.Assert.assertSame
import org.junit.Test

class VolumeAdaptionChangedEventTest {

    @Test
    fun volumeAdaptionSettingAndFeedIdAreStored() {
        val event =
            VolumeAdaptionChangedEvent(VolumeAdaptionSetting.LIGHT_REDUCTION, 3L)
        assertSame(VolumeAdaptionSetting.LIGHT_REDUCTION, event.volumeAdaptionSetting)
        assertEquals(3L, event.feedId)
    }
}
